### ISSUE TITLE
Window previews missing for applications in fullscreen mode

### ISSUE THREAD

**Issue by enkelprifti98** [steventheworker/DockAltTab#14]
I'm running into an issue where if an app is in fullscreen mode, the window preview in the dock does not show.

Is it possible to add support for fullscreen applications?

**Comment by steventheworker** [comment]
Update your [scriptable AltTab](https://github.com/steventheworker/alt-tab-macos/releases) when the next DockAltTab update is released. The last shortcut's will be labeled "DockAltTab" (instead of "Shortcut 5"). AltTab "show windows 'all spaces'" will need to be chosen under "Shortcut 5" (at AltTab Preferences -> Controls).

Will reply & close when it's ready (soon-ish). This has been fixed for a while, but I added it in the middle of a large update. 🫠

Eventually it'll also work with "visible spaces" checked (by only applying that setting if you're on a fullscreen space).
